csrc/mmdeploy/backend_ops/tensorrt/batched_bev_nms/trt_batched_bev_nms.cpp

```diff
@@ -1,6 +1,4 @@
 // Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
-// modify from
```

Comment:
Don't remove the copy right or reference information.

Reply:
ok, I will add it